content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -407,7 +407,7 @@ rules:
   # objects is "pods"
   resources: ["pods"]
   verbs: ["get", "list", "watch"]
-- apiGroups: ["batch", "extensions"]
+- apiGroups: ["batch", "networking.k8s.io"]
```

Comment:
jobs don't exist in the networking.k8s.io group... if you want to use a multi-group example, `"events"` resources in the core `""` and `"events.k8s.io"` API groups would be a current example

Reply:
Oh! Thanks very much for pointing it out and providing a great example!

I have pushed a new commit, could you please check the correctness again? :bowing_man: